src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Utilities/FormattingRangeHelper.cs

```diff
@@ -368,6 +368,7 @@ private static bool IsSpecialContainingNode(SyntaxNode node)
                 node.Kind() == SyntaxKind.FixedStatement ||
                 node.Kind() == SyntaxKind.GetAccessorDeclaration ||
                 node.Kind() == SyntaxKind.SetAccessorDeclaration ||
+                node.Kind() == SyntaxKind.InitAccessorDeclaration ||
```

Comment:
📝 This method appears to be related to the automatic formatting that occurs on `}` or `;`. We should add a test which fails prior to this change, but succeeds after. Perhaps something like this?

```
class C
{
  int Value
  {
    get;
init; // <- verify that typing this ';' triggers indentation
  }
}
```


```
class C
{
  int Value
  {
    get;
init
{
} // <- verify that typing this '}' triggers indentation of all three lines
  }
}
```

Reply:
@sharwell I looked at the current tests, but the way they're written makes it a bit hard for me to test this scenario.
Would be glad if you can take a look:

https://github.com/Youssef1313/roslyn/blob/patch-39/src/Workspaces/CoreTest/UtilityTest/FormattingRangeHelperTests.cs